.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -33,8 +35,15 @@ jobs:
         uses: hadolint/hadolint-action@v3.1.0
         with:
           dockerfile: app/Dockerfile
-          format: json
+          format: tty
           failure-threshold: warning
+          output-file: hadolint-results.txt
+
+      - name: Save output to workflow summary
+        if: always() # Runs even if there is a failure
+        run: |
+          echo "# Hadolint Results" >> $GITHUB_STEP_SUMMARY
```

Comment:
These `echo "# <job> Results"` are the markdown titles that I think we can actually remove now that I'm seeing how it looks

Reply:
Yeap, makes sense. Just pushed that for the jobs, I'll merge it later today if there is nothing else

Reply:
Nothing else, looks great, thanks!